lib/generators/ratyrate/templates/jquery.raty.js.erb

```diff
@@ -724,8 +724,8 @@
     cancel       : false,
     cancelClass  : 'raty-cancel',
     cancelHint   : 'Cancel this rating!',
-    cancelOff    : '<%= asset_path('cancel-off.png') %>',
-    cancelOn     : '<%= asset_path('cancel-on.png') %>',
+    cancelOff    : 'cancel-off.png',
```

Comment:
@devonlambert I think you should use asset_path here, otherwise it will not uses the asset pipeline to render the image